frontend/packages/kubevirt-plugin/src/selectors/vm/devices.ts

```diff
@@ -9,18 +11,27 @@ export const getBootDeviceIndex = (devices, bootOrder) =>
 export const getDeviceBootOrder = (device, defaultValue?): number =>
   device && device.bootOrder === undefined ? defaultValue : device.bootOrder;
 
-const getBootableDevices = (vm: VMKind): BootableDeviceType[] => {
-  const disks = getDisks(vm)
-    .filter((disk) => disk.bootOrder)
-    .map((disk) => ({ type: DEVICE_TYPE_DISK, value: disk }));
-  const nics = getInterfaces(vm)
-    .filter((nic) => nic.bootOrder)
-    .map((nic) => ({ type: DEVICE_TYPE_INTERFACE, value: nic }));
+export const getDevices = (vm: VMLikeEntityKind): BootableDeviceType[] => {
+  const disks = getDisks(asVM(vm)).map((disk) => ({
+    type: DEVICE_TYPE_DISK,
+    typeLabel: DiskWrapper.initialize(disk)
+      .getType()
+      .toString(),
+    value: disk,
+  }));
+  const nics = getInterfaces(asVM(vm)).map((nic) => ({
+    type: DEVICE_TYPE_INTERFACE,
```

Comment:
can we convert these into plain string enum?

Reply:
sure :+1: